components/Hero/Hero.jsx

```diff
@@ -0,0 +1,28 @@
+import styles from '@/styles/Hero.module.css'
+import Image from 'next/image'
+
+const Hero = (props) => {
+    const { title, description, src, alt, width, height } = props
```

Comment:
Please use an object for the image, i.e.: image: { src, width ... }

Reply:
passing image object as props